packages/mockserver-config-writer/src/mockserver-config/package-json.ts

```diff
@@ -81,6 +85,26 @@ function copyStartScript(startScript: string | undefined): string | undefined {
     return startMockScript;
 }
 
+/**
+ * Check if package.json has devDependency to @ui5/cli version  > 2.
+ *
+ * @param devDependencies - parsed devDependencies from package.json
+ * @returns - true: @ui/cli higher version 2; false: @ui/cli
+ */
+function isUi5CliHigherTwo(devDependencies: Partial<Record<string, string>>): boolean {
+    let isHigherTwo = false;
+    try {
+        const versionString = devDependencies['@ui5/cli'];
+        if (typeof versionString === 'string') {
+            const majorVersion = parseInt(versionString.split('.')[0].match(/\d+/)?.[0] || '0', 10);
+            isHigherTwo = majorVersion > 2 ? true : false;
```

Comment:
Could use `semver` module here
https://www.npmjs.com/package/semver

Reply:
Please use the `semver` module, we already use it in multiple places.

Reply:
I know about `semver`, and did consider using it. But adding `semver` adds other transient dependencies and in total 312K to the footprint of this module:

```shell
$ du -h node_modules 
 24K	node_modules/lru-cache
 48K	node_modules/semver/ranges
8.0K	node_modules/semver/bin
 32K	node_modules/semver/classes
 24K	node_modules/semver/internal
 96K	node_modules/semver/functions
252K	node_modules/semver
 32K	node_modules/yallist
312K	node_modules
```
I've decided, that is too much for replacing 1 line of code. If there would be more version comparison I'd agree and switch to it. Keep in mind, this module might be often called through `@sap-ux/create` with `npx` or `npm init` with on-the-fly installation, so it adds download and temp install time to each execution.

@devinea and @tobiasqueck , what do you think?

Reply:
Fair enough. 👍